interfaces/builtin/desktop.go

```diff
@@ -117,13 +125,58 @@ dbus (send)
 deny /{dev,run,var/run}/shm/lttng-ust-* rw,
 `
 
+var desktopFontconfigDirs = []string{
+	"/usr/share/fonts",
+	"/usr/local/share/fonts",
+	"/var/cache/fontconfig",
```

Comment:
I think these need to be set in `dirs`, so they inherit the global root dir.

Also, how cross-distro are these locations?

Reply:
Also, what happens if a different base snap doesn't have the mount point location available? Does it fail gracefully, or does it break the mounting sequence?  @zyga?

Reply:
I'll have a look at defining these through the `dirs` package.

The locations are fairly consistent.  `/usr/share/fonts` is fontconfig's default font location.  The `/usr/local/share/fonts` directory isn't part of upstream fontconfig, but is commonly configured by major distros:

https://sources.debian.net/src/fontconfig/2.12.3-0.2/debian/rules/#L20
https://src.fedoraproject.org/rpms/fontconfig/blob/master/f/fontconfig.spec#_69
https://build.opensuse.org/package/view_file/openSUSE:Factory/fontconfig/fontconfig.spec?expand=1
https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/fontconfig#n44

The fontconfig cache directory follows autoconf's `$localstatedir` directory, which is generally always configured as `/var` on Linux.

As for base snaps, I can see two solutions: (1) require base snaps to create these mount points in the same way they'd create `/etc`, `/home`, etc, or (2) rely on the upcoming layouts feature to ensure the mount points are available.